use-cases.md

```diff
@@ -69,11 +65,7 @@ The `operation` above refers to the name the server wants to give to the Span. F
 The `span` object above can be `None` if the Tracer did not find relevant headers in the incoming request: presumably because the client did not send them. In this case the server needs to start a brand new trace.
 
 ```python
-    span = tracer.join_trace_from_text(
-        operation_name=operation,
-        context_snapshot=request.headers,
-        trace_attributes=request.headers
-    )
+    span = tracer.get_span_extractor(request.class).extract(request)
```

Comment:
you guys both lost me at first with `StringMapExtractor`, now Ben's comment highlights exactly my concern - end user cannot write any Extractors, the encoding details are private to the tracer.